### PR TITLE
IGAPP-550: Add georgian and georgian fonts

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by klinzo** [comment]
Testable here: https://malteapp.de/nrw/ka
Fonts only added for web.

**Review by Taggotty — APPROVED** [review]
Looks fine 👍🏼

File: release-notes/unreleased/IGAPP-550-add-georgian.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-550
+show_in_stores: false
+platforms:
+  - web
+  - android
+  - ios
+en: Support added for georgian language and alphabet.
```
Comment (SteffiStuffel) [inline release-notes/unreleased/IGAPP-550-add-georgian.yml:7]: I think it is only for web, isn't it?
Why no german release note?

**Review by SteffiStuffel — APPROVED** [review]
It does not render the correct font when i start the app?

Bildschirmfoto zu 2021-04-22 12-10-18

File: release-notes/unreleased/IGAPP-550-add-georgian.yml (hunk 1 above)
Comment (klinzo) [reply release-notes/unreleased/IGAPP-550-add-georgian.yml:7]: See answer in other PR, if `show_in_stores` is set to true, it is not shown to users and therefore no german version is needed. I don't want to show this to users as there is only one city (and that is not even live and a malteser one) that uses this new language and as a user I would really be confused why this language is not available in my city anyway.

Regarding only web: It is native as well, we just dont have an additonal font there. But the language (should) work on native as well, which it did not previously to this issue.

**Comment by klinzo** [comment]
@SteffiStuffel I first thought it is because raleway (our decorative font) does not support Georgian, which is the case actually. But it uses Noto for me anyway? Could you refresh and try again please?
image

**Comment by Taggotty** [comment]
> @SteffiStuffel I first thought it is because raleway (our decorative font) does not support Georgian, which is the case actually. But it uses Noto for me anyway? Could you refresh and try again please?

You forgot to add the font to `I18nProvider.js`. After adding it works for me :)

**Comment by klinzo** [comment]
> > @SteffiStuffel I first thought it is because raleway (our decorative font) does not support Georgian, which is the case actually. But it uses Noto for me anyway? Could you refresh and try again please?
> 
> You forgot to add the font to `I18nProvider.js`. After adding it works for me :)

Thanks, did not know I had to add it there. Do you know by any chance why it was working for me anyway?

**Comment by Taggotty** [comment]
> Do you know by any chance why it was working for me anyway? 

We still set the font-family to Noto-Sans-Georgian in Layout.js . Therefore, I think the browser checks whether the System can provide this font and if yes it will use this one. If you by by any chance have the font installed on the system it will use this one, I guess.

**Pull request merged**